tests/cupy_tests/fft_tests/test_fft.py

```diff
@@ -127,6 +163,93 @@ def test_ifft(self, xp, dtype):
         return out
 
 
+# Almost identical to the TestFft class, except that
+# 1. multi-GPU cuFFT is used
+# 2. the tested parameter combinations are adjusted to meet the requirements
+@testing.parameterize(*testing.product({
+    'n': [None, 0, 64],
+    'shape': [(64,), (4, 64)],
+    'norm': [None, 'ortho', ''],
+}))
+@testing.with_requires('numpy>=1.10.0')
+@testing.multi_gpu(2)
+class TestMultiGpuFft(unittest.TestCase):
+    @multi_gpu_config(gpu_configs=[[0, 1], [1, 0]])
+    @testing.for_complex_dtypes()
+    @testing.numpy_cupy_allclose(rtol=1e-4, atol=1e-7, accept_error=ValueError,
+                                 contiguous_check=False)
+    def test_fft(self, xp, dtype):
+        a = testing.shaped_random(self.shape, xp, dtype)
+        out = xp.fft.fft(a, n=self.n, norm=self.norm)
+
+        # np.fft.fft alway returns np.complex128
+        if xp == np and dtype is np.complex64:
```

Comment:
`xp is np` seems more natural.

Reply:
That's a good catch. I didn't think carefully when copying. I correct all such occurrences throughout this test suite.

Reply:
Could you fix `xp == cupy` too?

Reply:
I don't understand. When `xp is cupy`, the returned type is `complex64`. What do you want to fix?

Reply:
@asi1024?

Reply:
Sorry to confuse you. I commented it here because it is related to `xp is np`, but you don't need to fix this line anymore. https://github.com/cupy/cupy/pull/2644/files#r358111247

Reply:
Ah, OK. More legacy code to change...Thanks for catching them.